web/styles/informational_overlays.css

```diff
@@ -1,108 +1,186 @@
 .informational-overlays {
+    display: grid;
+    place-items: center;
+
     .overlay-content {
-        /* because zoom breaks at 525px perhaps due to rounding errors, so add a
-           trivial amount of width so it doesn't break. */
-        width: 550px;
+        width: 90%;
+        max-width: 650px;
         margin: 0 auto;
         position: relative;
-        top: calc((30vh - 50px) / 2);
-        border-radius: 4px;
+        top: unset;
+        border-radius: 6px;
         overflow: hidden;
 
         background-color: var(--color-background-modal);
     }
 
     .overlay-tabs {
-        padding: 10px 0;
-        border-bottom: 1px solid hsl(0deg 0% 0% / 20%);
+        position: relative;
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        gap: 15px;
+        padding: 0 5%;
+        height: 60px;
+        box-shadow: 0 3px 12px 1px hsl(207deg 100% 11% / 8%);
+        z-index: 2;
 
         .tab-switcher {
-            margin-left: 15px;
+            flex-grow: 1;
         }
 
         .exit {
-            float: right;
-            font-size: 1.5rem;
-            color: hsl(0deg 0% 67%);
-            font-weight: 600;
-            margin: 0 15px;
+            display: inline-block;
+            font-size: 16px;
+            line-height: 1em;
+            text-align: center;
+            width: 1em;
+            color: hsl(0deg 0% 60%);
             cursor: pointer;
+            order: 1;
         }
     }
 
-    .overlay-modal {
-        padding-bottom: 10px;
+    .modal-body {
+        height: 75vh;
+        max-height: unset;
+        text-align: center;
+        outline: none;
+    }
+
+    .simplebar-content {
+        padding: 35px 30px 35px 25px !important;
+    }
+
+    & a {
+        color: var(--color-text-link);
 
-        .modal-header h3 {
-            font-weight: 300;
+        &:hover {
+            color: var(--color-text-link) !important;
         }
+    }
 
-        .modal-body {
-            height: 70vh;
-            text-align: center;
-            outline: none;
+    .documentation-link {
+        text-align: center;
+        margin-top: 35px;
+    }
+
+    & kbd {
+        font-family: "Source Sans 3", sans-serif;
+        font-feature-settings: "ss01" on;
+        margin: 6px 6px 0;
+        padding: 1px 0.5em;
+        font-size: 14px;
+        font-weight: 400;
+        color: var(--color-keyboard);
+        background: none;
+        border: solid 1.5px var(--color-keyboard);
+        border-radius: 5px;
+        text-shadow: none;
+    }
+
+    .concurrent-hotkeys {
+        display: inline-block;
```

Comment:
(Though is looks like you'd need to make some adjustments to how hotkeys are presented under "The Basics" section if you move to flexbox; that would probably mean adjusting some HTML.)

Reply:
I changed it to flex as you suggested and it is cleaner/less complex as not/first/last-child are now not required. I think you were suggesting an HTML change because the shortcuts with a `or` in between were extending to the next line and breaking CSS, right? ( like this: 
![image](https://github.com/zulip/zulip/assets/64723994/69e40bcf-1538-42f9-8725-e2c9f58ebe09))

I fixed this by using `display: inline-flex` and It is not breaking CSS and is giving the same design as before, can you check if this is the right approach?

Reply:
Yeah, in this case, `inline-flex` is probably the better approach. The issue seems to be the text nodes that we get with things like "or" that don't have markup of their own.